app/models/concerns/date_helpers.rb

```diff
@@ -0,0 +1,13 @@
+module DateHelpers
+
+  private
+
+  def valid_date?(date)
+    Date.parse(date.to_s)
+  rescue ArgumentError
+    return false
+  else
```

Comment:
You mean like this?
```ruby
def valid_date?(date)
   return Date.parse(date.to_s)
 rescue ArgumentError
   return false
end
```

Or even better `Date.parse(date.to_s) && return true`  ?